test/suites/operations.sh

```diff
@@ -0,0 +1,62 @@
+test_operations_get_all() {
+  assert_get_all_operations() {
+    (
+      set -e
+      echo "==> get all-projects operations normally"
+      result="$(lxc query "/1.0/operations?all-projects=true" | jq '.success[0]' | wc -l)"
+      test "${result}" -eq 1 || false
+
+      echo "==> get all-projects operations recursively"
+      result="$(lxc query "/1.0/operations?recursion=1&all-projects=true" | jq '.success[0]' | grep -c 'status_code')"
```

Comment:
```suggestion
      result="$(lxc query "/1.0/operations?recursion=1&all-projects=true" | jq '[.success[] | select(.status_code)] | length')"
```
This will count the number of occurrences of `status_code`.